[3.2] docs(textbox): Revamped TextBox Overview (#830)
telerik/blazor-docs: components/textbox/overview.md

> You can control [various attributes](#features) of the `input` element and turn the Textbox into a password box, for example. You can also configure this component to respond to [events]({%slug components/textbox/events%}).

Reviewer: ```suggestion
You can control [various attributes and behaviors](#textbox-parameters) of the `input` element, for example turn the component to a password textbox. The component also responds to [events]({%slug components/textbox/events%}).
```

}
````

## Appearance

The TextBox component provides settings to control its appearance. [Read more about the Blazor TextBox appearance settings]({%slug textbox-appearance%}).

## Events

> The Blazor TextBox generates blur and value change events for further customizing its behavior. [Read more about the Blazor TextBox events]({%slug components/textbox/events%}).

Reviewer: ```suggestion
The Blazor TextBox emits **blur** and value **change** events for further customizing its behavior. [Read more about the Blazor TextBox events]({%slug components/textbox/events%}).
```

| `Name` | `string` | The `name` attribute of the HTML element. It is usually required so the `AutoComplete` will be honored by the browser. |
| `Password` | `bool` | When set to `true`, the HTML element renders `type="password"` so that the user input is hidden. You can find examples of validation and reveal buttons in the [Live Demo: Password Textbox](https://demos.telerik.com/blazor-ui/textbox/password) |
| `PlaceHolder` | `string` | A `string` that maps to the `placeholder` attribute of the HTML element. If a `Label` is defined, it will be shown instead of the placeholder when the input is not focused. |
| `TabIndex` | ``Nullable<int>`` | Maps to the `tabindex` attribute of the HTML element. You can use it to customize the order in which the inputs in your form focus with the `Tab` key. |
| `Title` | `string` | Maps to the `title` attribute of the HTML element. You can use it to add a [tooltip]({%slug tooltip-overview%}). |

> | `ValidateOn` | `ValidationEvent` | Configures the event that will trigger validation (if validation is enabled). Read more at [Validation Modes for Simple Inputs]({%slug common-features/input-validation%}#validation-modes-for-simple-inputs). |

Reviewer: What is the default value here? All custom types (enums) that we have should specify it.
